StarWars/StarWars.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Linq;
+
+public class Program
+{
+    public static void Main()
+    {
+        bool QuitRoster = false;
+        Person leia = new Person("Leia", "Organa", "Rebel");
+        Person han = new Person("Han", "Solo", "Rebel");
+        Person luke = new Person("Luke", "Skywalker", "Jedi");
+        Person yoda = new Person("Yoda", "", "Jedi");
+        Ship falcon = new Ship("Rebel", "Smuggling", 2, "Millenium Falcon");
+        Ship tie = new Ship("Tie", "Fighter", 2, "Tie Fighter");
+        Ship xwing = new Ship("X-Wing", "Fighter", 1, "Imperial Fighter");
+        Station rebelss = new Station("Rebel Space Station", "Rebel", 5);
+        Station deathstar = new Station("Death Star", "Imperial", 10);
+
+        //Enter in crew to ships
+        try
+        {
+            falcon.EnterShip(leia, 0);
+            falcon.EnterShip(han, 1);
+            tie.EnterShip(luke, 0);
+            tie.EnterShip(yoda, 1);
+        }
+        catch (System.IndexOutOfRangeException e)
```

Comment:
Good use of exceptions. A better place for this though, would be in the EnterShip method. That way you always know when something is wrong vs having to remember what that exception means.